js/skin.js

```diff
@@ -152,8 +161,9 @@ var Skin = React.createClass({
    * @param pageY {number} y coordinate
    * @returns {[number, number, number]}
    */
-
   getDirectionParams: function(pageX, pageY) {
+    pageX = Utils.ensureNumber(pageX) ? pageX : 0;
```

Comment:
The ternary operator is not necessary when using ` Utils.ensureNumber()`. The function takes a second parameter which is a default value to use when the input is not a number. This can be simplified like this: `Utils.ensureNumber(pageX, 0)`

Reply:
\+